cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/ConsentRespExt.kt

```diff
@@ -130,3 +131,23 @@ internal fun Map<String, Map<String, Boolean>>.toAcceptedCategories(): Iterable<
     val falseCategories = partitions.second.map { it.first }.toSet()
     return trueCategories.minus(falseCategories)
 }
+
+fun JSONObject.getStringOrNullByKey(
```

Comment:
Using a `try{ .. } catch(e){ return null }` is an anti-pattern in java, use the `isNull` api instead